src/cli/lint.ts

```
@@ -0,0 +1,114 @@
+import type { Options } from '@cspell/eslint-plugin'
+import cspellRecommended from '@cspell/eslint-plugin/recommended'
+import js from '@eslint/js'
+import react from '@eslint-react/eslint-plugin'
+import { logger } from '@rspress/shared/logger'
```

ESLint

[error] Unable to resolve path to module '@rspress/shared/logger'.

(import-x/no-unresolved)